src/enemy.cairo

```diff
@@ -4,8 +4,14 @@ use cubit::types::Fixed;
 #[derive(Component, Serde, Drop)]
 struct Enemy {
```

Comment:
Lets switch this to using the `Vehicle` component, since they will share much of their logic. I've made this change in the sensors branch already